Kernel/CMakeLists.txt

```diff
@@ -350,6 +350,7 @@ if ("${SERENITY_ARCH}" STREQUAL "x86_64")
         ${KERNEL_SOURCES}
         Arch/x86_64/init.cpp
 
+        Arch/PageFault.cpp
```

Comment:
Should shared files be up here https://github.com/SerenityOS/serenity/pull/16911/commits/d641bdcb33858ffd85d5d9c035176c1b98582d77#diff-cea8f745e8a814046fe796d65a88932274144e595d7b94351ad52549f484f7aeR18 once instead of being listed once in the x86_64 and once in the aarch64 section?

(also for Processor.cpp I suppose, but that shouldn't change in this commit at least.)

Reply:
Yep you are correct, but I think eventually we don't want any cpp files in the `Arch/` directory, however for now that is easier as the `PageFault.h` file is also in that directory. I could also move them to the `Kernel/` directory if you really want to. Nevertheless I moved the `Arch/PageFault.cpp` line in the `CMakelists.txt` to the common list.